python/lsst/cell_coadds/_identifiers.py

```diff
@@ -32,7 +32,7 @@
 from dataclasses import dataclass
 from typing import TYPE_CHECKING, Union
 
-from lsst.skymap import CellInfo, Index2D, PatchInfo
+from lsst.skymap import CellInfo, Index2D, Optional, PatchInfo
```

Comment:
Although I can't reproduce this so are we sure `Optional` really is being imported and this code is running?

Reply:
Nope, we are not running anything (yet). Which is also why I left a generic comment saying we might have to fix bugs when we start running this. The PR is so that I could work in parallel on other parts of the package and finally get it running.

Reply:
And no, importing `Optional` from `lsst.skymap` didn't work for me either.

Reply:
The tests running in GHA also indicate that this is broken, so at least that's working.